code/game/area/areas.dm

```diff
@@ -292,24 +292,23 @@ var/global/list/areas = list()
 
 var/global/list/mob/living/forced_ambiance_list = new
 
-/area/Entered(A)
-	if(!istype(A,/mob/living))	return
-
-	var/mob/living/L = A
-	if(!L.ckey)	return
+/area/Entered(mob/living/L, old_loc)
+	if(istype(L) && L.ckey)
```

Comment:
Doesn't this make non-player mobs immune to gravity changes?

Reply:
Looks like they always were, reading the old code. Can fix.